### PR TITLE
Drop tables

### PR DESCRIPTION
**Pull request by ChrisBr**
we already have datatables. It doesn't make sense to distinguish between tables and datatables
they should look similar IMHO.

This is a starting point for a discussion, I'm open for other ideas / suggestions.

cc @openSUSE/open-build-service @mdeniz

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit 4826f0d48add05be75abf5d451f9c6065a792a25

https://deploy-preview-25--obs-patterns.netlify.com

**Comment by dmarcoux** [comment]
Fine for me, but the datatable page needs to be updated. We need to add the sorting and more columns to clearly show how it reacts on small screens. I'm probably forgetting something.

**Comment by ChrisBr** [comment]
> Fine for me, but the datatable page needs to be updated. 

Yep, I just wanted to get some more input first...

**Review by bgeuken — APPROVED** [review]
Fine with me. Just update the data table component before merging, as requested by Dany:smile:

**Comment by saraycp** [comment]
Fine for me as well.

**Comment by ChrisBr** [comment]
@dmarcoux updated, please have another look

**Comment by coolo** [comment]
\<super_joke>Wouldn't it be easier to just drop the database?</super_joke>

**Comment by ChrisBr** [comment]
> <super_joke>Wouldn't it be easier to just drop the database?</super_joke>

not funny

File: _posts/components/2018-08-31-datatable.html [hunk 1]
```diff
@@ -1,12 +1,38 @@
 ---
 layout: post
-title: DataTable
+title: Table
```
Comment (dmarcoux) [inline _posts/components/2018-08-31-datatable.html]: The file should be renamed to use `table`, not `datatable`.

**Review by dmarcoux — DISMISSED** [review]
There is an error in the changes, Netlify cannot process them.

**Comment by ChrisBr** [comment]
> There is an error in the changes, Netlify cannot process them.

Will look into it

**Comment by dmarcoux** [comment]
@ChrisBr You can run the project locally with the Docker setup. It will be easier to catch what is going wrong.

**Comment by dmarcoux** [comment]
I fixed the issue. Review again please.

**Comment by ChrisBr** [comment]
> I fixed the issue. Review again please.

Thanks :heart:

**Pull request merged**